sys-libs/glibc/glibc-9999.ebuild

```diff
@@ -79,11 +78,10 @@ fi
 #     (unexported) is used to sanity check compatibility before enabling
 #     debugging.
 #     Also see https://sourceware.org/gdb/wiki/FAQ#GDB_does_not_see_any_threads_besides_the_one_in_which_crash_occurred.3B_or_SIGTRAP_kills_my_program_when_I_set_a_breakpoint
-#   * normal 'strip' command trims '.symtab'
 #   Thus our main goal here is to prevent 'libpthread.so.0' from
 #   losing it's '.symtab' entries.
-# As Gentoo's strip does not allow us to pass less aggressive stripping
-# options and does not check the machine target we strip selectively.
+#   * normal 'strip' command trims '.symtab' but "--strip-debug" keeps .symtab.
+export PORTAGE_STRIP_FLAGS="--strip-debug -N __gentoo_check_ldflags__ -R .comment"
```

Comment:
Thanks,

This is based on GDB docs suggestion to use "--strip-debug with libpthread.so .
https://sourceware.org/gdb/wiki/FAQ#GDB_does_not_see_any_threads_besides_the_one_in_which_crash_occurred.3B_or_SIGTRAP_kills_my_program_when_I_set_a_breakpoint

I do not have a strong opinion which approach is better.  In Chrome OS, we decided to use "--strip-debug" for glibc since that allows us to ship a stripped "libpthread.so" on devices saving some precious image size. So I was trying to reduce divergence between glibc ebuild between Gentoo and Chrome OS.

I def agree if there was a clean way in portage to specify strip options on a particular file, that'd be the best solution.